bin/picca_coadd_zint.py

```diff
@@ -0,0 +1,458 @@
+        'name': 'NT',
+        'value': headers_to_check_match_values['NT'],
+        'comment': 'Number of bins in r-transverse'
+    },
+    {
+        'name': 'ZCUTMIN',
+        'value': z_cut_min,
+        'comment': 'Minimum redshift of pairs'
+    },
+    {
+        'name': 'ZCUTMAX',
+        'value': z_cut_max,
+        'comment': 'Maximum redshift of pairs'
+    },
+    {
+        'name': 'NSIDE',
+        'value': headers_to_check_match_values['NSIDE'],
+        'comment': 'Healpix nside'
+    },
+    {
+        'name': 'OMEGAM',
+        'value': headers_to_check_match_values['OMEGAM'],
+        'comment': 'Omega_matter(z=0) of fiducial LambdaCDM cosmology'
+    },
+    {
+        'name': 'OMEGAR',
+        'value': headers_to_check_match_values['OMEGAR'],
+        'comment': 'Omega_radiation(z=0) of fiducial LambdaCDM cosmology'
+    },
+    {
+        'name': 'OMEGAK',
+        'value': headers_to_check_match_values['OMEGAK'],
+        'comment': 'Omega_k(z=0) of fiducial LambdaCDM cosmology'
+    },
+    {
+        'name': 'WL',
+        'value': headers_to_check_match_values['WL'],
+        'comment': 'Equation of state of dark energy of fiducial LambdaCDM cosmology'
+    }
+    ]
+    results.write(
+        [r_par, r_trans, z, num_pairs],
+        names=['RP', 'RT', 'Z', 'NB'],
+        comment=['R-parallel', 'R-transverse', 'Redshift', 'Number of pairs'],
+        units=['h^-1 Mpc', 'h^-1 Mpc', '', ''],
+        header=header,
+        extname='ATTRI')
+
+    header2 = [{
+        'name': 'HLPXSCHM',
+        'value': 'RING',
+        'comment': 'Healpix scheme'
+    }]
+    results.write([healpixs, weights, xi],
+                  names=['HEALPID', 'WE', 'DA'],
+                  comment=['Healpix index', 'Sum of weight', 'Correlation'],
+                  header=header2,
+                  extname='COR')
+
+    results.close()
+
+    return
+
+def coadd_dmats(fi,fout):
+    """Coadds distortion matrices measured in different redshift intervals.
+
+    Args:
+        fi: list of strings
+            List of the paths to the input distortion matrices to be coadded.
+        fout: string
+            String of the path to the desired output.
+    """
+
+    # specify which header entries we want to check are consistent across all
+    # files being coadded
+    headers_to_check_match = ['NP','NT','COEFMOD','REJ','OMEGAM','OMEGAR','OMEGAK','WL']
+
+    # initialize distortion matrix array, fill them with zeros
+    with fitsio.FITS(fi[0]) as hdul:
+        dmat = hdul[1]['DM'][:] * 0
+        weights_dmat = hdul[1]['WDM'][:] * 0
+        r_par_dmat = np.zeros(hdul[2]['RP'][:].size)
+        r_trans_dmat = np.zeros(hdul[2]['RT'][:].size)
+        z_dmat = np.zeros(hdul[2]['Z'][:].size)
+        num_pairs_dmat = 0.
+
+        # get values of header entries to check with other files
+        header = hdul[1].read_header()
+        headers_to_check_match_values = {h:header[h] for h in headers_to_check_match}
+
+    ### OLD WEIGHTS
```

Comment:
I think it'd be best to remove these comments before merging

Reply:
Agreed, I'll address this in a commit shortly